melos.yaml

```diff
@@ -54,15 +54,6 @@ scripts:
       dirExists: lib
       dependsOn: "build_runner"
 
-  slang:
-    run: dart run slang
-    exec:
-      concurrency: 1
-    description: Run generate translation code.
-    packageFilters:
-      dirExists: lib
-      dependsOn: slang
-
```

Comment:
I switched the positions because it seemed easier to read with `gen` and `slang` grouped separately!